client/src/components/Monument/Monument.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Monument.css';
+
+export default class Monument extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {monument: {}, monumentProperties: []};
+    }
+
+    async componentDidMount() {
```

Comment:
Why is this method called `componentDidMount()`? Is this a standard or something?

Reply:
Yes, it's part of the component lifecycle. This basically is overriding `React.Component`'s method, since this class extends that one